sktime/forecasting/scinet.py

```diff
@@ -0,0 +1,286 @@
+        self.input_len_seg = input_len_seg
+        self.positionalE = positionalE
+        self.modified = modified
+        self.RIN = RIN
+
+        super().__init__(
+            num_epochs=num_epochs,
+            batch_size=batch_size,
+            in_channels=in_channels,
+            individual=individual,
+            criterion_kwargs=criterion_kwargs,
+            optimizer=optimizer,
+            optimizer_kwargs=optimizer_kwargs,
+            lr=lr,
+        )
+
+        from sktime.utils.dependencies import _check_soft_dependencies
+
+        if _check_soft_dependencies("torch"):
+            import torch
+
+            self.criterions = {
+                "MSE": torch.nn.MSELoss,
+                "L1": torch.nn.L1Loss,
+                "SmoothL1": torch.nn.SmoothL1Loss,
+                "Huber": torch.nn.HuberLoss,
+            }
+
+            self.optimizers = {
+                "Adadelta": torch.optim.Adadelta,
+                "Adagrad": torch.optim.Adagrad,
+                "Adam": torch.optim.Adam,
+                "AdamW": torch.optim.AdamW,
+                "SGD": torch.optim.SGD,
+            }
+
+    def _build_network(self, fh):
+        # Define the SCINet-based network
+        from sktime.networks.scinet import SCINet
+
+        return SCINet(
+            seq_len=self.seq_len,
+            input_dim=self._y.shape[-1],
+            pred_len=fh,
+            hid_size=self.hid_size,
+            num_stacks=self.hid_size,
+            num_levels=self.num_levels,
+            num_decoder_layer=self.num_decoder_layer,
+            concat_len=self.concat_len,
+            groups=self.groups,
+            kernel=self.kernel,
+            dropout=self.dropout,
+            single_step_output_One=self.single_step_output_One,
+            input_len_seg=self.input_len_seg,
+            positionalE=self.positionalE,
+            modified=self.modified,
+            RIN=self.RIN,
+        )
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return ``"default"`` set.
+
+
+        Returns
+        -------
+        params : dict or list of dict
+        """
+        params = [
+            {
+                "seq_len": 8,
+                "pred_len": 1,
+                "lr": 0.005,
+                "optimizer": "Adam",
+                "batch_size": 1,
+                "num_epochs": 1,
+                "individual": True,
+            },
+            {
+                "seq_len": 16,
+                "pred_len": 2,
+                "lr": 0.005,
+                "optimizer": "Adam",
+                "batch_size": 1,
+                "num_epochs": 1,
+                "individual": False,
+            },
+        ]
```

Comment:
as said above, remove individual if not used

keep the 2 test params different from each other, you can initialize 2 different architectures from the config.